integrations/tensorflow/e2e/math/BUILD

```diff
@@ -0,0 +1,958 @@
+    "bessel_i1e",
+    "betainc",
+    "bincount",
+    "confusion_matrix",
+    "cosh",
+    "count_nonzero",
+    "cumprod",
+    "cumsum",
+    "cumulative_logsumexp",
+    "digamma",
+    "divide",  # Failing for integer inputs because iree doesn't output 'f64'.
+    "erf",
+    "erfc",
+    "erfinv",
+    "expm1",
+    "igamma",
+    "igammac",
+    "in_top_k",
+    "invert_permutation",
+    "is_nan",
+    "is_non_decreasing",
+    "is_strictly_increasing",
+    "l2_normalize",
+    "logical_and",
+    "logical_not",
+    "logical_or",
+    "logical_xor",
+    "ndtri",
+    "nextafter",
+    "polygamma",
+    "pow",
+    "reduce_all",
+    "reduce_any",
+    "reduce_euclidean_norm",
+    "reduce_logsumexp",
+    "reduce_prod",
+    "reduce_std",
+    "reduce_variance",
+    "rint",
+    "segment_max",
+    "segment_mean",
+    "segment_min",
+    "segment_prod",
+    "segment_sum",
+    "sign",
+    "sobol_sample",
+    "softsign",
+    "unsorted_segment_max",
+    "unsorted_segment_mean",
+    "unsorted_segment_min",
+    "unsorted_segment_prod",
+    "unsorted_segment_sqrt_n",
+    "unsorted_segment_sum",
+    "xdivy",
+    "xlog1py",
+    "xlogy",
+    "zeta",
+]
+
+# ---- INDIVIDUAL STATIC TESTS ----------------------------------------------- #
+
+# These tests allow us to generate coverage tables and give a finer-grained view
+# of the coverage, but are very slow due to bazel overhead, so they are not
+# run on the internal or OSS CI.
+iree_e2e_cartesian_product_test_suite(
+    name = "math_tests",
+    srcs = ["math_test.py"],
+    failing_configurations = [
+        {
+            # Failing on TFLite.
+            "functions": TFLITE_FAILING,
+            "target_backends": "tflite",
+        },
+        {
+            # Failing on vmla.
+            "functions": VMLA_FAILING,
+            "target_backends": "iree_vmla",
+        },
+        {
+            # Failing on llvm.
+            "functions": LLVM_FAILING,
+            "target_backends": "iree_llvmjit",
+        },
+        {
+            # Failing on vulkan.
+            "functions": VULKAN_FAILING,
+            "target_backends": "iree_vulkan",
+        },
+    ],
+    flags_to_values = {
+        "reference_backend": "tf",
+        "functions": TF_MATH_FUNCTIONS,
+        "dynamic_dims": False,
+        "test_complex": False,
+        "target_backends": [
+            "tf",
+            "tflite",
+            "iree_vmla",
+            "iree_llvmjit",
+            "iree_vulkan",
+        ],
+    },
+    main = "math_test.py",
+    tags = [
+        "manual",
+        "nokokoro",
+        "notap",
+    ],
+    deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+        "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+    ],
+)
+
+# ---- MULTIPLE STATIC TESTS ------------------------------------------------ #
+
+# These tests compile all functions in tf.math at once for testing so that
+# we can run them on the CI with 5 additional targets instead of 640.
```

Comment:
I guess they'd become out of date if we `tf.math` adds more functions and we add them too? I included failing targets here so it won't change otherwise.